results/mysql-kafka/README.md

```diff
@@ -0,0 +1,93 @@
+# MySQL-to-Kafka results
```

Comment:
Let's move the readme and charts into `/results/mysql-kafka/20250507`, as they are related to the run on that date. If we ever make some changes and re-run the tests we'd create a similar folder with a newer date and a separate report and charts.

Note: watch out for the relative links in this file, they'll need to be updated.

Reply:
done